.github/workflows/validate.yaml

```diff
@@ -39,6 +39,22 @@ jobs:
         with:
           path: ./src/github.com/${{ github.repository }}
 
+      - name: Merge upstream
+        if: github.event_name == 'pull_request'
+        with:
+          path: ./src/github.com/${{ github.repository }}
+        run: |
+          if ! git config user.name > /dev/null; then
+            git config user.name "John Doe"
```

Comment:
Could use something like: "GitHub Actions"

Reply:
This is not a commit we will see in the history, so it doesn't really matter

Reply:
Yeah just a suggestion to make it look more related anyway.